commit 44aa9319ea920f12e139cb00f6c9df9529f6bb0a
Author: Adrian Knoth <adi@drcomp.erfurt.thur.de>
Date:   Sun Jan 6 20:34:56 2013 +0100

    wscript: Set HAVE_OPUS=0 if custom headers are missing
    
    As reported by Peter Nelson, waf's check_cfg sets HAVE_OPUS=1 if the
    pkg-config check is successful, however, the code requires OPUS custom
    headers.
    
    So if a dev package without custom headers is installed, explicitly
    reset HAVE_OPUS to 0 to fix the FTBFS.

diff --git a/wscript b/wscript
index 568dcf6..02d59c3 100644
--- a/wscript
+++ b/wscript
@@ -179,6 +179,8 @@ def configure(conf):
         if conf.check_cc(header_name='opus/opus_custom.h', mandatory=False):
             conf.define('HAVE_OPUS', 1)
             conf.env['WITH_OPUS'] = True
+        else:
+            conf.define('HAVE_OPUS', 0)
 
 
     conf.env['LIB_PTHREAD'] = ['pthread']
